fedimint-server/src/consensus/aleph_bft/keychain.rs

```diff
@@ -15,6 +16,7 @@ pub struct Keychain {
     peer_id: PeerId,
     public_keys: BTreeMap<PeerId, PublicKey>,
     keypair: KeyPair,
+    tag: sha256::Hash,
```

Comment:
@joschisan I don't want to block your PRs, but consensus code is like the most sophisticated, important part of our codebase with least amount of understanding coverage among contributors, so while you're improving it, please, please add a comment to every struct and field you're touching. At least the non-trivial ones, or at least `struct`s. It  pays off big time when you're the reader trying to figure out something from the codebase.

Here, I'm opening a PR, I see a "tag", and I have no idea what I am looking at. Had the `Keychain` had a one sentence docstring, it would make understanding it much easier.

Reply:
I agree, renamed and added a comment. I actually have no further changes to core consensus planned at the moment, so agree with you here.